FirebaseVertexAI/Sources/Chat.swift

```diff
@@ -115,12 +135,11 @@ public class Chat {
         }
 
         // Save the request.
-        history.append(contentsOf: newContent)
+        appendHistory(contentsOf: newContent)
 
         // Aggregate the content to add it to the history before we finish.
-        let aggregated = aggregatedChunks(aggregatedContent)
-        history.append(aggregated)
-
+        let aggregated = self.aggregatedChunks(aggregatedContent)
+        self.history.append(aggregated)
```

Comment:
Would this need to be something like?
```suggestion
        appendHistory(contentsOf: aggregated)
```

Reply:
Good catch. Done.

Reply:
It doesn't look like the diff made it in the last commit

Reply:
Oh, it's the difference between appending one array element versus appending another array. 

I believe the first should be atomic (since it will use the setter) and the second may not be.

Reply:
Ah, I see. Sorry for the misdirection. I didn't realize `self.history.append(aggregated)` was atomic. You are correct. I did a quick test in playgrounds and it looks atomic.